### PR TITLE
Polygon generation from script

### PR DESCRIPTION
**Pull request by nenau-again**
Ok, for some reason that I don't understand, github refuses to make a PR to naev/naev-atwork. Can you take the file from here ?

### CODE REVIEW

**Comment by ids1024** [comment]
> Ok, for some reason that I don't understand, github refuses to make a PR to naev/naev-atwork

You'll need to re-clone `naev-artwork` and add the file. @bobbens suggested using [git LFS](https://git-lfs.github.com/), and I migrated the repo to that, which required editing all the git history.

File: 3D/polygon/polygon_from_sprite.py [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+    mydata = mydata.toprettyxml(indent="\t",encoding="UTF-8")
+
+    myfile = open(adress, "w")
+    myfile.write(mydata.decode("utf-8") )
+    myfile.close()
+
+
+if __name__ == "__main__":
+    #pntNplg = polygonFromPng('../../../naev/dat/gfx/ship/admonisher/admonisher.png',8,8,150,3,6)
+    pntNplg = polygonFromPng('../../../naev/dat/gfx/outfit/space/caesar.png',6,6,150,2,4)
```
Comment (ids1024) [inline 3D/polygon/polygon_from_sprite.py:435]: Having to manually edit this each time a polygon is generated, and tune the parameters is counterproductive to the aim I stated in naev#11.

Could the script be modified to include the right parameters for each ship and be able to generate them all?

**Comment by nenau-again** [comment]
I'll do that.
There is just a problem with `autocannon.png`, that has strange pixels here and there. For this one, I had to manually disable the refining process. It could be solved if we edit directly the png and erase the meaningless pixels. Would that be right ?

**Comment by nenau-again** [comment]
Transferred to naev/naev-artwork

**Pull request closed without merging**